doc/specs/lichess-api.yaml

```diff
@@ -58,6 +58,13 @@ info:
     - `curl https://lichess.org/api/account -H "Authorization: Bearer {token}"`
     - [NodeJS example](https://github.com/lichess-org/api/tree/master/example/oauth-personal-token)
 
+    ### Token Security
+    - Keep your tokens secret. Do not share them in public repositories or public forums.
```

Comment:
Maybe would add something like "Your tokens can be used to make your account perform arbitrary actions (within the limits of the tokens' scope). You remain responsible for all activities on your account." here or as a separate point above or below.